core/node/options_transactor.go

```diff
@@ -21,5 +21,5 @@ package node
 type OptionsTransactor struct {
 	TransactorEndpointAddress string
 	RegistryAddress           string
-	AccountantID              string
+	ChannelImplementation     string
```

Comment:
ChannelImplementation field naming is not clear, is it some kind of standard? Maybe it can be named ChannelID?

Reply:
ChannelImplementation address is an address in blockchain where channel implementation code is hosted.

All deployed channels are pointing into this smart contract. So it's definitely not ChannelID.